app/client/components/SpeedIndicator.js

```diff
@@ -0,0 +1,39 @@
+
+import React from 'react';
+
+export default class SpeedIndicator extends React.Component {
+
+    static propTypes = {
+        speed: React.PropTypes.number.isRequired,
+    }
+
+    static defaultProps = {
+        speed: 'wer',
+    }
+
+    render() {
+        var { speed } = this.props;
+
+        var speedStyle = {
+          backgroundColor: 'red',
+          width: '30px',
```

Comment:
when it come to `px` you should just give in a number and React will do the magic:

```
var stuyle = {
  width: 22,
};
```